cmd/fixer.go

```diff
@@ -503,102 +191,25 @@ var contentMismatch64Kfixer = &cobra.Command{
 		}
 
 		domain := args[0]
-		corruptedSuffix := "-corrupted"
-
-		if !noDryRunFlag {
-			fmt.Println("This is a dry-run, no file will be altered")
+		q := url.Values{
+			"dry_run": {strconv.FormatBool(!noDryRunFlag)},
 		}
 
 		c := newAdminClient()
 		res, err := c.Req(&request.Options{
-			Method: "GET",
-			Path:   "/instances/" + url.PathEscape(domain) + "/fsck",
+			Method:  "GET",
+			Path:    "/instances/" + url.PathEscape(domain) + "/content-mismatch-fixer",
```

Comment:
I'd prefer to have a namespace for fixers: `"/instances/" + url.PathEscape(domain) + "/fixers/content-mismatch"`